partcad/src/partcad/converters/step_converter.py

```
@@ -0,0 +1,23 @@
+        if status != 1:
+            raise RuntimeError(f"Failed to read STEP file: {self.input_file}")
+        step_reader.TransferRoot()
+        return step_reader.Shape()
+
+    def write(self, shape: TopoDS_Shape) -> None:
+        """Write a TopoDS_Shape to a STEP file."""
+        step_writer = STEPControl_Writer()
+        step_writer.Transfer(shape, STEPControl_AsIs)
+        status = step_writer.Write(str(self.output_file))
+        if status != 1:
+            raise RuntimeError(f"Failed to write STEP file: {self.output_file}")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)